### PR TITLE
Create node instance

### PR DESCRIPTION
**Pull request by nktsitas**

### CODE REVIEW

**Review by papazach — APPROVED** [review]
👍🏼  You could also add a new line in the end of the file just make the warning go away

**Review by chrikar — APPROVED** [review]

**Review by lvrach — APPROVED** [review]

File: proto/node/create/v1/node_instance_creation.proto [hunk 1]
```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
```
Comment (lvrach) [inline proto/node/create/v1/node_instance_creation.proto:1]: We can move the file in: `proto/nodeinstance/v1/creation.proto`, in order to be consistent with other files.

File: proto/nodeinstance/create/v1/creation.proto [hunk 2]
```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+option go_package = "node_instance/creation/v1;node_instancecreation";
+
+package nodeinstance.create.v1;
+
+message CreateNodeInstance {
+    string claim_id = 1;
```
Comment (underhood) [inline proto/nodeinstance/create/v1/creation.proto:7]: can we add comments to ensure claim_id of agent sending or noted being created.

Comment (nktsitas) [reply proto/nodeinstance/create/v1/creation.proto:10]: Added couple comments Timo, LMK if this covers what you had in mind :)

**Review by underhood — APPROVED** [review]

**Pull request merged**